examples/rosettacode/rock_paper_scissor.nit

```diff
@@ -0,0 +1,56 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+#
+# Task: Programming_Task/Rock-paper-scissors
+# SEE: <http://rosettacode.org/wiki/Rock-paper-scissors>
+#
+# author: Istvan szalai 22/05/2015
+
+# Return a value to dertermine who win the match.
+# Return 0 for lose, 1 for win, 2 for equality.
+fun win_or_lose(player, computer: String): Int do
+	if player == computer then return 2
+	if player == "rock" and computer == "scissors" then return 1
+	if player == "rock" and computer == "paper" then return 0
+	if player == "paper" and computer == "rock" then return 1
+	if player == "paper" and computer == "scissors" then return 0
+	if player == "scissors" and computer == "paper" then return 1
+	return 0
+end
+
+# Return a string with about the match winner or looser.
```

Comment:
`with about` ? This does not make sense.

You should probably just say that the method returns whether the match gives a winner or a loser.

Or better then, since the number is exclusively 0, 1 or 2, you could have a literal array access in your main on the right string, hence this method could disappear altogether